linux/system/profile.sls

```diff
@@ -0,0 +1,30 @@
+{%- from "linux/map.jinja" import system with context %}
+{%- if system.enabled %}
+
+{%- if system.profile.get('enabled', False) %}
+# on /etc/profile (system wide, no variable expansion)
+
+{%- if system.profile.get('proxy', {}).get('enabled', False) %}
```

Comment:
No, this should be generic class with metadata like:
```
linux:
  system:
    profile:
      proxies:
         enabled: true
         script: |
          export HTTP_PROXY=blahblah
```
resulting in file `/etc/profile.d/proxies.sh` etc.